examples/controlplane/controlplane-single/controlplane.yaml

```diff
@@ -0,0 +1,115 @@
+#####################################################
+# ${CLUSTER_NAME}-controlplane-0
+#####################################################
+apiVersion: cluster.x-k8s.io/v1alpha2
+kind: Machine
+metadata:
+  name: ${CLUSTER_NAME}-controlplane-0
```

Comment:
not sure keep this -0 is still useful, ok to keep it

Reply:
It would make it more consistent if the cluster is Lager upgraded to multi-node. Also the diff is smaller to multi so its easier to compare

Reply:
ok, works for me